EXE/subtitles.xml

```diff
@@ -187,43 +190,43 @@
 <Static AudioFile="mod\m04\m04_offer_0090_trent.wav" TextArea="Cinematic">See you there.</Static>
 
 
-<Static AudioFile="echo_m04.utf|0xAAD01AC4" TextArea="Cinematic">This is commander Marcus Creitmire of the Hessian Lions spec ops unit.</Static>
-<Static AudioFile="echo_m04.utf|0x96D304C4" TextArea="Cinematic">Turn your weapon systems off immediately, and follow our lead. If you try to resist, we have the authority to destroy you.</Static>
+<Static AudioFile="echo_m04.utf|0xAAD01AC4" TextArea="Cinematic">This is commander Marcus Creitmire of the Hessian Lions special operations unit.</Static>
+<Static AudioFile="echo_m04.utf|0x96D304C4" TextArea="Cinematic">Power down your weapon systems immediately, and follow our lead. If you try to resist, we have the authority to destroy you.</Static>
```

Comment:
"Power your weapon systems down" or "Power down your weapons" without "systems"

Reply:
Oh, Of course! They are German!
Thank you.